libp2p/Host.cpp

```diff
@@ -763,11 +791,14 @@ void Host::startedWorking()
         m_nodeTable = nodeTable;
     restoreNetwork(&m_restoreNetwork);
```

Comment:
I just realized that `restoreNetwork` requires `m_true` to be `true`, too, let's fix it in another PR

Reply:
Ah thanks for the clarification